### PR TITLE
Add `.DS_Store` and `xcuserdata` to .gitignore

### PR DESCRIPTION
**Pull request by gui-dos**

### CODE REVIEW

**Comment by gui-dos** [comment]
I am noticing that DEVELOPMENT_TEAM and PRODUCT_BUNDLE_IDENTIFIER defined in `LibreWristConfig.xcconfig` are unused in many points of the `project.pbxproj` and of the entitlements files.

This PR should ease a lot the building process to potential contributors like me... ;-)

File: LibreWristWatch Watch App/LibreWristWatch Watch App.entitlements
Comment (poml88) [inline LibreWristWatch Watch App/LibreWristWatch Watch App.entitlements:1]: But here we have to add the variable <string>$(APP_GROUP_ID)</string> ?

Comment (gui-dos) [reply LibreWristWatch Watch App/LibreWristWatch Watch App.entitlements:1]: I tried but if I keep it LibreWrist doesn't build for me (I am on Xcode 16.1 beta 3).

Comment (poml88) [reply LibreWristWatch Watch App/LibreWristWatch Watch App.entitlements:1]: Ok? Strange.
Well, the shared group is needed to share the username password auth_token with the complications...

Comment (gui-dos) [reply LibreWristWatch Watch App/LibreWristWatch Watch App.entitlements:1]: A curiosity: why are you using the double “group.group” prefix?

Comment (gui-dos) [reply LibreWristWatch Watch App/LibreWristWatch Watch App.entitlements:1]: You are right: ce095e9 just reassigns the group `(APP_GROUP_ID)` defined in `.xcconfig`.

**Review by poml88 — APPROVED** [review]
Thanks!

**Comment by poml88** [comment]
@gui-dos Question: How do I do with the ##DEV_TEAM##? I have to set my team, but then, when I commit something the placeholder is gone... Should the config file be in the .gitignore?

**Comment by gui-dos** [comment]
@poml88

Personally, I adhere to the same conventions as other DIY projects such as Glucose Direct, xDrip4iOS, iAPS, ...

In fact, my `DiaBLE.xcconfig` was actually submitted by FreeAPSX’s member @mountrcg (<https://github.com/ninghaixin/DiaBLE/commit/47640fd>). The technique is to allow the user to customize the variable substitutions in a `*Override.xcconfig` file (added manually or downloaded online) which is invoked at the end by the standard one in the repo but that is itself ignored by adding for example a `*Override.xcconfig` line in `.gitignore`.

**Comment by poml88** [comment]
Ok, thanks, yes, I have seen this override thingy... Guess have to find out how that works.

**Pull request merged**